web/src/beta/components/PublishStateSwitchField/index.tsx

```diff
@@ -0,0 +1,116 @@
+import { useState } from "react";
+
+import Icon from "@reearth/beta/components/Icon";
+import Text from "@reearth/beta/components/Text";
+import { styled, useTheme } from "@reearth/services/theme";
+
+const PublishStateSwitchField: React.FC = () => {
+  const labels = ["Unpublished", "Published"];
```

Comment:
Please use i18n internationalization for text. Check classic components for examples 👍🏻